packages/css/src/components/link-list/link-list.template.ts

```diff
@@ -20,20 +20,28 @@ function ol(children: TemplateResult) {
   `;
 }
 
-export function linkListTemplate({ type, links }: LinkList) {
+export function linkListTemplate({ navLabel, type, links }: LinkList) {
   const children = html`
     ${links.map(link => html`
       <li>
-        ${anchorTemplate(link)}
+        ${anchorTemplate(navLabel ? link : { ...link, ariaCurrent: undefined })}
```

Comment:
Dit ziet er wat vunzig uit. Kijkend naar of het organisme een `navLabel` heeft, kill je van een molecuul een eigenschap. Eigenlijk wijst dit erop dat hoe dan ook deze mengvorm niet helemaal lekker is. Op zich laten we zien wat we moeten zien, maar het voelt niet zo goed. @tfrijsewijk wat denk jij?

Reply:
Blijkbaar komt dit stukje code al uit jouw koker @tfrijsewijk, hetgeen mijn vraag beantwoordt: we laten zien wat we moeten laten zien, en hoe we daartoe komen mag soms ondergeschikt zijn aan het doel.

Reply:
Deze change geeft ons een relatie tussen `ariaCurrent` en `navLabel`: Zonder `navLabel` zal `ariaCurrent` altijd `undefined`.

De relatie is gedocumenteerd EN geforceerd: we kunnen nooit (per ongeluk) een link list met `ariaCurrent` maar zonder `navLabel` maken.

Dit is een silent fix. Ik heb er nog over nagedacht om boven regel 24 een check op `navLabel && links.some(a => a.ariaCurrent)` met `throw new Error()`. Kunnen we ook doen, wat zijn jullie voorkeuren?

Reply:
Nou en kijk eens:

![image](https://user-images.githubusercontent.com/9696261/159458672-4ce75a21-81ed-45fe-8277-8222d392e0e0.png)

Vallen we toch zo door de mand met z'n allen.. @daverhelst. Zie https://github.com/dso-toolkit/dso-toolkit/pull/1473/files#diff-5e30b937afa553e072777bc6c3ba1d9c20e77fffc1361f793e1ac6647ed6bba7R21.

Reply:
Goed gezien! Dit is inderdaad een overblijfsel van een eerdere poging. Dan zou zo'n `throw new Error()` echt wel wenselijk geweest zijn, om onszelf en de codebase schoon en netjes te houden.

Voor nu heb ik dit object uit de content gehaald.